### PR TITLE
Packs lose gossip functions into service, requester or broadcaster

### PR DESCRIPTION
**Pull request by luca-moser**

### CODE REVIEW

File: pkg/protocol/gossip/requester.go [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+	for _, f := range r.backPFuncs {
+		if f() {
+			return true
+		}
+	}
+	return false
+}
+
+// AddBackPressureFunc adds a RequestBackPressureFunc.
+// This function can bbe called multiple times to add additional RequestBackPressureFunc.
```
Comment (muXxer) [inline pkg/protocol/gossip/requester.go]: typo "bbe"

File: pkg/tangle/solidifier.go [hunk 2]
```diff
@@ -327,7 +327,7 @@ func (t *Tangle) solidifyMilestone(newMilestoneIndex milestone.Index, force bool
 		}
 	}()
 
-	t.log.Infof("Run solidity check for Milestone (%d)...", milestoneIndexToSolidify)
+	t.log.Infof("RunRequestQueueDrainer solidity check for Milestone (%d)...", milestoneIndexToSolidify)
```
Comment (muXxer) [inline pkg/tangle/solidifier.go]: :wutpepe:

Comment (luca-moser) [reply pkg/tangle/solidifier.go]: :karlos:

File: pkg/protocol/gossip/requester.go (hunk 1 above)
Comment (luca-moser) [reply pkg/protocol/gossip/requester.go]: bibi blocksberg

**Review by muXxer — APPROVED** [review]

**Pull request merged**